src/sentry/api/endpoints/organization_group_index.py

```diff
@@ -99,6 +100,12 @@ def get(self, request, organization):
         if not project_ids:
             return Response([])
 
+        try:
+            start, end = get_date_range_from_params(request.GET)
+        except InvalidParams as exc:
+            return Response({'detail': exc.message}, status=400)
+
+        # TODO(jess): check these event / short id results against date filter
```

Comment:
will do this tomorrow, either in this pr or in a follow up -- just want to unblock frontend work asap

Reply:
If we get a short id and date options, I think the short ID should take priority.

Reply:
works for me. same for event id or do you think that should respect date filters?

Reply:
I would generally favour the more precise input. event id and short-id are very specific keys and should take precedence over range selectors imo.